agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -313,6 +314,17 @@ void main_thread::after_select(bool timeout)
             clear_ready(soc->sta_hal_int_events);
         }
     }
+
+    // Send topology discovery every 60 seconds according to IEEE_Std_1905.1-2013 specification
+    static std::chrono::steady_clock::time_point discovery_timestamp =
+        std::chrono::steady_clock::now();
+    auto now     = std::chrono::steady_clock::now();
+    auto elapsed = std::chrono::duration_cast<std::chrono::seconds>(now - discovery_timestamp);
+    if ((m_eFSMState == EState::CONNECTED) ||
+        (m_eFSMState == EState::OPERATIONAL && elapsed.count() == 60)) {
+        discovery_timestamp = now;
```

Comment:
As usual with repeating timers, the question is: should we accumulate the jitter (like you do here) or should we approximate the rate as much as possible (by doing `discovery_timestamp += 60` instead). But in the latter case, you also have to make sure that you handle multiple deadline misses (e.g. with `if (now - discovery_timestamp >= 60) discovery_timestamp = now`). But again, all of that would lead us a bit too far. Maybe add a TODO.